funnel/views/session.py

```diff
@@ -102,13 +103,21 @@ class SessionView(SessionViewMixin, UrlForView, ModelView):
     @route('')
     @render_with('schedule.html.jinja2', json=True)
     def view(self):
-        return dict(project=self.obj.project, active_session=session_data(self.obj, with_modal_url='view_popup'),
-            from_date=date_js(self.obj.project.schedule_start_at), to_date=date_js(self.obj.project.schedule_end_at),
-            sessions=session_list_data(self.obj.project.scheduled_sessions, with_modal_url='view_popup'),
+        project_save_form = SavedProjectForm()
+        project_currently_saved = self.obj.project.is_saved_by(current_auth.user)
+        return {
+            'project': self.obj.project,
+            'active_session': session_data(self.obj, with_modal_url='view_popup'),
+            'from_date': date_js(self.obj.project.schedule_start_at),
+            'to_date': date_js(self.obj.project.schedule_end_at),
+            'sessions': session_list_data(self.obj.project.scheduled_sessions, with_modal_url='view_popup'),
             # FIXME: This timezone by UTC offset is not accounting for DST. Look up where it's being used and fix it
-            timezone=utcnow().astimezone(self.obj.project.timezone).utcoffset().total_seconds(),
-            venues=[venue.current_access() for venue in self.obj.project.venues],
-            rooms=dict([(room.scoped_name, {'title': room.title, 'bgcolor': room.bgcolor}) for room in self.obj.project.rooms]))
+            'timezone': utcnow().astimezone(self.obj.project.timezone).utcoffset().total_seconds(),
```

Comment:
The `FIXME` can be resolved if we replace `utcnow()` with `project.schedule_start_at` (if present, falling back to `utcnow`). Also, the parameter should be called `tzoffset`, not `timezone`. Templates need a corresponding update @vidya-ram.